traveler-ios-ui/TravelerKitUI/ViewControllers/BookableAvailabilityViewController.swift

```diff
@@ -96,8 +96,8 @@ extension BookableAvailabilityViewController: BookingContextObserving {
     }
 }
 
-extension BookableAvailabilityViewController: BookableConfirmationViewControllerDelegate {
-    func bookableConfirmationViewControllerDidConfirm(_ controller: BookableConfirmationViewController, bookingForm: BookingForm) {
-        delegate?.bookableAvailabilityViewControllerDidConfirm(self, bookingForm: bookingForm)
+extension BookableAvailabilityViewController: BookablePassesViewControllerDelegate {
+    func bookablePassesViewControllerDidReceiveCheckout(withForm form: BookingForm?) {
```

Comment:
Booking form shouldn't be optional. Change method signature (See comments above)